src/oscar/apps/dashboard/catalogue/views.py

```diff
@@ -65,20 +66,7 @@
 Option = get_model('catalogue', 'Option')
 
 
-def filter_products(queryset, user):
```

Comment:
Could you please leave this function in place for now, and apply the [deprecated](https://github.com/django-oscar/django-oscar/blob/master/src/oscar/core/decorators.py#L6) decorator to it? This will warn any projects that rely on it that it will be removed in the next version of Oscar.

Reply:
That's a good idea. I have done that. Please check the commits now.